doc/rst/language/spec/classes.rst

```diff
@@ -205,11 +204,11 @@ The memory management strategies have the following meaning:
 -  ``owned`` the instance will be deleted automatically when the
    ``owned`` variable goes out of scope, but only one ``owned`` variable
    can refer to the instance at a time. See the module documentation for
-   :mod:`owned <OwnedObject>`.
+   :record:`~OwnedObject.owned`.
 
 -  ``shared`` will be deleted when all of the ``shared`` variables
    referring to the instance go out of scope. See
-   the module documentation for :mod:`shared <SharedObject>`.
+   the module documentation for :record:`~SharedObject.shared`.
```

Comment:
@mppf: Would you be willing to review the changes to this part of the spec?  Essentially, the changes here were:
* we don't want SharedObject/OwnedObject to show up in the module index
* squashing them from the index also squashes the ability to cross-reference them
* however, that seems appropriate given that we really want them to be part of the spec / not known to the user as modules anyway
As a result, I anchored these to the record type declarations instead.  Other tricks could be used to link to other parts of the document (though I'm imagining that this may all get rejiggered once we bring these docs into the spec anyway...).

Reply:
LGTM